src/main/groovy/com/marklogic/gradle/task/roxy/CopyRoxyFilesTask.groovy

```diff
@@ -0,0 +1,53 @@
+package com.marklogic.gradle.task.roxy
+
+import com.marklogic.gradle.task.MarkLogicTask
+import org.apache.commons.io.FileUtils
+import org.gradle.api.tasks.TaskAction
+
+import static groovy.io.FileType.FILES
+
+class CopyRoxyFilesTask extends MarkLogicTask {
+
+	def roxyFolderMapping = [
+								"src" : "src/main/ml-modules/root",
+								"rest-api/config" : "src/main/ml-modules/options",
+								"rest-api/ext" : "src/main/ml-modules/services",
+								"rest-api/transforms" : "src/main/ml-modules/transforms"
+							]
+
+	@TaskAction
+	void copyRoxyFiles() {
+		backupProperties()
```

Comment:
This should be part of the "else", so that backup files aren't made unless the user provided valid input.